tests/common/playground.rs

```diff
@@ -184,15 +192,15 @@ default-collection = \"{}\"
                         continue;
                     }
                 }
-                RetryExpectations::ExpectNoOut => {
-                    if out.out.is_empty() {
-                        return;
-                    } else {
-                        println!("Expected no out but was {}", out.out);
-                        sleep(interval);
-                        continue;
-                    }
-                }
+                // RetryExpectations::ExpectNoOut => {
```

Comment:
Why commented out?

Reply:
It wasn't being used any where on main, but now that I've moved it, the compiler is picking it up as dead code. But I thought we might use it in the future, so I commented it out instead of removing.

Reply:
I'm a bit confused why this code isn't used anymore.

Reply:
I also am, but I don't think that it's gone from being used to not. It never appears to have been used but I think the re-arrange of where the tests are now means that it is flagged as dead code.